packages/content-type-remote-attachment/README.md

```diff
@@ -134,7 +136,8 @@ await conversation.messages.send(remoteAttachment, {
 });
 ```
 
-Note that we’re using `contentFallback` to enable clients that don't support these content types to still display something. For cases where clients *do* support these types, they can use the content fallback as alt text for accessibility purposes.
+> **Note**  
```

Comment:
See line 135 - I think this might need to be updated to reflect the content fallback from the codec?

Reply:
yep! we can remove L#135